config/opensearch_dashboards.yml

```diff
@@ -281,6 +281,13 @@
 #    'ff00::/8',
 # ]
 
+# Optional setting that enables you to specify a path to PEM files for the certificate
+# authority for your connected datasources.
+#data_source.ssl.certificateAuthorities: [ "/path/to/your/CA.pem" ]
+
+# To disregard the validity of SSL certificates for connected data sources, change this setting's value to 'none'.
+#data_source.ssl.verificationMode: full
```

Comment:
do we support only two verificationMode [full, none] or more ?

Reply:
kk find three support modes [none, full, certificate], maybe let's add more description in the yml entry comment

Reply:
Added possible values as a comment in this file.